tests/e2e/commons_test.go

```diff
@@ -18,7 +18,7 @@ package e2e
 
 import "github.com/cloudnative-pg/cloudnative-pg/tests/utils"
 
-func GetEnvProfile() utils.EnvProfile {
+func MustGetEnvProfile() utils.EnvProfile {
```

Comment:
I think we really need to refactor this whole file, the other function can panic too but from the name is not obvious